packages/gafl-webapp-service/src/handlers/__tests__/page-handler.spec.js

```diff
@@ -12,7 +12,9 @@ jest.mock('../../constants', () => ({
   ANALYTICS: {
     selected: 'selected',
     acceptTracking: 'accepted-tracking',
-    seenMessage: 'seen-message'
+    seenMessage: 'seen-message',
+    skipPage: 'skip-page',
+    pageSkipped: 'page-skipped'
```

Comment:
if you're mocking these, they should be different to the values in constants, otherwise there's a risk of stuff being tightly bound